public/ext/kablammo/data/sequenceserver_blast_result20150201-27830-1cjljf8.xml

```diff
@@ -0,0 +1,2558 @@
+<?xml version="1.0"?>
```

Comment:
Is this a sample result or committed by mistake? This needs to be removed. It's confusing.